README.md

```diff
@@ -8,6 +8,17 @@ Please see [psyclone.pdf](psyclone.pdf) in this directory (or on
 more information. If you would prefer to build the documentation,
 please see the [README](doc/README.md) file in the "doc" directory.
 
+# Try it on Binder #
+
+Some of the examples are available as Jupyter notebooks. These may
+be launched using Binder from the links below.
+
+ * Existing Fortran code: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/575_jupyter_egs?filepath=examples%2Fnemo%2Feg4%2Fcopy_stencil.ipynb)
```

Comment:
This first example is perhaps a bit out-of-data? You show the psy and the kern parts of the example but the psy part already shows the kern part so it is seen twice. Is that what you meant to do? I also think we need to have information saying what is going on at each step - but that can wait for a subsequent PR.

Reply:
I think it's OK as a first step. The bit with the kernel is really (I think) to demonstrate `walk()` - I've added comments.